### ISSUE TITLE
Journeys not being called

### ISSUE THREAD

**Issue by picolloo** [elastic/synthetics#332]
Hi there, 

I'm implementing synthetics using the heartbeat Docker image but my journey is not being called by the elastic module. 
All my files are being placed in `/usr/share/heartbeat/suites`, with name `xxxx.journey.js` but they are not executed. 

My manifest file look like this:
```yml
heartbeat.monitors:
- type: browser
  name: Accounts
  id: accounts-auth
  schedule: "@every 1m"
  source:
    local:
      path: "${SUITES_DIR}"
```

docker-compose.yml:
```yml
version: '3.2'

services:
  heartbeat:
    build:
      context: .
    command: --strict.perms=false -c /etc/heartbeat.yml -e
    env_file:
     - .env
    volumes:
      - type: bind
        source: ./manifests/development/heartbeat.yml
        target: /etc/heartbeat.yml
      - "./dist:/usr/share/heartbeat/suites"
    networks:
      - docker-elk_elk

networks:
  docker-elk_elk:
    external: true

```

I'm unable to deploy my synthetics tests because of it. Am I doing anything wrong?

**Comment by picolloo** [comment]
I'm receiving this message from the docker logs:

```bash
heartbeat_1  | 2021-06-30T20:18:17.447Z INFO    browser/browser.go:39   Synthetic monitor detected! Please note synthetic monitors are an experimental unsupported feature!
heartbeat_1  | 2021-06-30T20:18:17.716Z INFO    synthexec/synthexec.go:128      Running command: /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3 in
 directory: '/tmp/elastic-synthetics-046899047'
heartbeat_1  | 2021-06-30T20:18:17.933Z INFO    synthexec/synthexec.go:224      stderr: error: unknown option '--json'
heartbeat_1  | 2021-06-30T20:18:17.936Z ERROR   synthexec/synthexec.go:172      Error waiting for command /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --o
utfd 3: exit status 1
heartbeat_1  | 2021-06-30T20:18:17.936Z INFO    synthexec/synthexec.go:176      Command has completed(1): /tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --o
utfd 3
heartbeat_1  | 2021-06-30T20:18:17.936Z WARN    synthexec/synthexec.go:183      Error executing command '/tmp/elastic-synthetics-046899047/node_modules/.bin/elastic-synthetics . --screenshots --json --network --ou
tfd 3': exit status 1
```

**Comment by vigneshshanmugam** [comment]
@picolloo `--json` flag was removed by mistake in our latest release of synthetics agent which is why you are running to the issue. For now, Since you are running suites, you can control the synthetics version and use the synthetic version `v1.0.0-beta.3`.

**Comment by vigneshshanmugam** [comment]
Published a new version that fixes the issue, `v1.0.0-beta.6` Please update your package.json and let us know if that works for you.

**Comment by picolloo** [comment]
Thank for the fast response @vigneshshanmugam, I bumped to version beta.6 but now I'm receiving this error:

```bash
heartbeat_1  | 2021-07-01T11:01:24.610Z INFO    browser/browser.go:39   Synthetic monitor detected! Please note synthetic monitors are an experimental unsupported feature!
heartbeat_1  | 2021-07-01T11:01:24.986Z INFO    synthexec/synthexec.go:128      Running command: /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3 in directory: '/tmp/elastic-synthetics-391316926'
heartbeat_1  | 2021-07-01T11:01:25.247Z INFO    synthexec/synthexec.go:224      stderr: error: unknown option '--network'
heartbeat_1  | 2021-07-01T11:01:25.250Z ERROR   synthexec/synthexec.go:172      Error waiting for command /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3: exit status 1
heartbeat_1  | 2021-07-01T11:01:25.250Z INFO    synthexec/synthexec.go:176      Command has completed(1): /tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3
heartbeat_1  | 2021-07-01T11:01:25.250Z WARN    synthexec/synthexec.go:183      Error executing command '/tmp/elastic-synthetics-391316926/node_modules/.bin/elastic-synthetics . --screenshots --json --network --outfd 3': exit status 1
```
